AxonDeepSeg/segment.py

```diff
@@ -187,19 +188,40 @@ def main():
     :return: None.
     '''
     print 'AxonDeepSeg v.{}'.format(AxonDeepSeg.__version__)
-    ap = argparse.ArgumentParser()
+    ap = argparse.ArgumentParser(formatter_class=RawTextHelpFormatter)
+
+    requiredName = ap.add_argument_group('required arguments')
 
     # Setting the arguments of the segmentation
-    ap.add_argument("-t", "--type", required=True, help="Choose the type of acquisition you want to segment.") # type
-    ap.add_argument("-i", "--imgpath", required=True, nargs='+', help="Folder where the images folders are located.")
-    ap.add_argument("-m", "--model", required=False,
-                    help="Folder where the model is located.", default=None)
-    ap.add_argument("-s", "--sizepixel", required=False, help="Pixel size in micrometers to use for the segmentation.",
-                    default=0.0)
-    ap.add_argument("-v", "--verbose", required=False, help="Verbosity level.", default=0)
-    ap.add_argument("-o", "--overlap", required=False, help="Overlap value when doing the segmentation. The higher the"
-                                                            "value, the longer it will take to segment the whole image.",
-                    default=25)
+    requiredName.add_argument("-t", "--type", required=True, choices=['SEM','TEM'], help="Type of acquisition to segment. \n"
+                                                                                        "SEM: scanning electron microscopy samples. \n"
+                                                                                        "TEM: transmission electron microscopy samples. ")
+    requiredName.add_argument("-i", "--imgpath", required=True, nargs='+', help="Path to the image to segment or path to the folder \n"
+                                                                                "where the image(s) to segment are located.")
```

Comment:
The usage is misleading because when indicating the path to the image directly, there is an error. See below:
~~~
axondeepseg -t TEM -i unnamed_23.87nm.jpg -s 0.02387
AxonDeepSeg v.0.3
Traceback (most recent call last):
  File "/Users/julien/ads_venv/bin/axondeepseg", line 11, in <module>
    load_entry_point('AxonDeepSeg', 'console_scripts', 'axondeepseg')()
  File "/Users/julien/code/axondeepseg/AxonDeepSeg/segment.py", line 258, in main
    verbosity_level=verbosity_level)
  File "/Users/julien/code/axondeepseg/AxonDeepSeg/segment.py", line 94, in segment_folders
    for file_ in tqdm(os.listdir(path_testing_images_folder), desc="Segmentation..."):
OSError: [Errno 20] Not a directory: 'unnamed_23.87nm.jpg'
~~~
So, either there is a bug in the code, or there is an error in the usage help.